cmd/start.go

```diff
@@ -59,7 +59,7 @@ func init() {
 		}
 }
 
-// Start starts a node; if it's not a bootstrap node it will get the Node.URLs
+// Start starts a node; if it's not a bootstrap node it will get the node.URLs
```

Comment:
Should still be `Node.URLs` because it's about the location in config file.

Reply:
Ah, my IDE's rename feature went too far.